src/main/java/com/lginterview/filter/ThrottlingFilter.java

```diff
@@ -0,0 +1,67 @@
+package com.lginterview.filter;
+
+import com.google.common.util.concurrent.RateLimiter;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Component
```

Comment:
[MS][Question] I think Filter should have explicit @Order annotation.

Reply:
Yes it should